The removed extensions are not packaged for Void and are not useful for
building the manual pages of interest for this package.

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -36,10 +36,6 @@
 # ones.
 extensions = [
     "breathe",
-    "myst_parser",
-    "sphinx_copybutton",
-    "sphinxcontrib.rsvgconverter",
-    "sphinxcontrib.moderncmakedomain",
 ]
 
 breathe_projects = {"pybind11": ".build/doxygenxml/"}
